lib/dao.js

```diff
@@ -636,7 +636,7 @@ DataAccessObject.upsert = function(data, options, cb) {
  * If there is a single instance retrieved, update the retrieved model.
  * Creates a new model if no model instances were found.
  * Returns an error if multiple instances are found.
- * * @param {Object} [where]  `where` filter, like
+ * @param {Object} [where]  `where` filter, like
```

Comment:
Second eyes help sometimes. You should remove the extra space in front of the `]`. ;)